src/sgame/sg_admin.cpp

```diff
@@ -2350,6 +2358,92 @@ bool G_admin_setlevel( gentity_t *ent )
 	return true;
 }
 
+bool G_admin_slap( gentity_t *ent )
+{
+	int            pid;
+	double         damage = 0.0f;
+	char           name[ MAX_NAME_LENGTH ], err[ MAX_STRING_CHARS ];
+	gentity_t      *vic;
+	vec3_t         dir;
+
+	if ( level.intermissiontime )
+	{
+		return false;
+	}
+
+	if ( trap_Argc() < 2 )
+	{
+		ADMP( QQ( N_("^3slap:^* usage: slap [^3name|slot#|admin#^7] (^3damage^7)") ) ); 
+		return false; // todo: G_admin_helptext( "slap" ) instead of this copypasta
```

Comment:
to introduce a new feature with a TODO usually means the TODO will never be fixed :)

Reply:
I need to speak to @DolceTriade about this, but basically I want to make a helper function which retrieves the help info automatically from the initial definition... there's several others I'd like to do but that's outside the scope of this PR.

Reply:
Sure, it would be useful, but agree about adding TODOs. Ideally discussions about what you want to do in the future happen in issues or IRC/discord.

If you want a PR, there should be no TODOs